src/matchers/toBeBefore/predicate.test.js

```diff
@@ -1,14 +0,0 @@
-import predicate from './predicate';
-
-const EARLIER = new Date('06/02/2018');
```

Comment:
_index.test.js_ has all the dates formatted in ISO 8601. So deleting this file deletes the only tests with this format. Not sure if we care about that.